proto/types/canonical.proto

```diff
@@ -29,9 +29,9 @@ message CanonicalProposal {
 
 message CanonicalVote {
   SignedMsgType             type      = 1;  // type alias for byte
-  int64                     height    = 2 [(gogoproto.moretags) = "binary:\" fixed64 \""];
-  int64                     round     = 3 [(gogoproto.moretags) = "binary:\" fixed64 \""];
-  CanonicalBlockID          block_id  = 5 [(gogoproto.nullable) = false, (gogoproto.customname) = "BlockID"];
-  google.protobuf.Timestamp timestamp = 6 [(gogoproto.nullable) = false, (gogoproto.stdtime) = true];
-  string                    chain_id  = 7 [(gogoproto.customname) = "ChainID"];
+  sfixed64                  height    = 2;  // canonicalization requires fixed size encoding here
+  sfixed64                  round     = 3;  // canonicalization requires fixed size encoding here
```

Comment:
@Liamsi what you think of reducing this to sfixed32. It will br breaking in the amount of bytes expected but I dont see a reason to keep 64 other than backwars compat

Reply:
I think 32 bytes for round are sufficient but that would break SignBytes (see https://github.com/tendermint/tendermint/pull/5008/files#r440061804)😬 
Hence, I'd leave it for encoding only.